src/commands/mfe/rm.ts

```diff
@@ -22,6 +25,48 @@ export default class Rm extends Command {
 
     const microFrontendService: MicroFrontendService =
       new MicroFrontendService()
+    const mfeReferences = microFrontendService.findWidgetConfigReferences(
+      args.name
+    )
+
+    if (mfeReferences.length > 0) {
+      const { removeConfigs } = await inquirer.prompt([
+        {
+          name: 'removeConfigs',
+          message: `Reference to ${args.name} will be removed from this bundle's microfrontend widgets. Do you wish to continue?`,
```

Comment:
Could you please replace "this bundle's microfrontend widgets" with the actual list of widgets names? We agreed that when we discussed the AC.